server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.ts

```diff
@@ -129,6 +129,18 @@ export class AgenticChatController implements ChatHandlers {
     #additionalContextProvider: AdditionalContextProvider
     #contextCommandsProvider: ContextCommandsProvider
 
+    /**
+     * Determines the appropriate message ID for a tool use based on tool type and name
+     * @param toolType The type of tool being used
+     * @param toolUse The tool use object
+     * @returns The message ID to use
+     */
+    #getMessageIdForToolUse(toolType: string | undefined, toolUse: ToolUse): string {
+        const toolUseId = toolUse.toolUseId!
+        // Return plain toolUseId for executeBash, add "_permission" suffix for all other tools
+        return toolUse.name === 'executeBash' || toolType === 'executeBash' ? toolUseId : `${toolUseId}_permission`
```

Comment:
I'm not sure I understand what this code is doing. Can we just do something like this:
```
if (toolUse.name === 'executeBash') {
       return toolUse.name
}
return `${toolUseId}_permission`
```